projects/step-core/src/lib/modules/automation-package-common/components/automation-package-filter-popover/automation-package-filter-popover.component.scss

```
@@ -12,6 +12,8 @@ mat-card {
   }
 
   mat-card-title {
-    font-size: 2rem;
+    font-size: 1.8rem;
+    padding: 16px;
```

```suggestion
    padding: 1.6rem;
```
We try to avoid px, 1rem = 10px

---

Sure, changed px to rem